pelux.xml

```diff
@@ -125,4 +125,10 @@
            name="Pelagicore/meta-smarcimx8m"
            path="sources/meta-smarcimx8m"/>
 
+  <project remote="github"
+           upstream="thud-l4t-r32.1"
+           revision="29677fe9484eddaa351dd0a05b0a8f8fb15fe075"
+           name="madisongh/meta-tegra"
+           path="sources/meta-tegra"/>
+
```

Comment:
Consider add attribute groups, like
`groups="notdefault,external,jetson"/>
`
to not include it by default. But perhaps we shall handle this as a separate issue since it may affect other BSP layers too, and the way of working with how to setup the build environment.

Reply:
Yes, it'll make CI/CD a bit more complicated, but I agree there is a room for an improvement.

Reply:
Usage
`repo init --groups <group>
`